src/key_scan.rs

```diff
@@ -372,11 +373,17 @@ pub fn receive_history_entry<L: Ledger>(
     };
 
     let txn_asset = last_record.asset_def.code;
-    TransactionHistoryEntry {
+    TransactionParams::<L> {
+        uid: Some(uid),
+        timeout: None,
+        status: TransactionStatus::Unknown,
+        memos: Default::default(),
+        sig: None,
+        inputs: Default::default(),
+        outputs: Default::default(),
```

Comment:
I wonder if this should be `records`, since we actually have access to the record openings? Could be an eventual GUI feature, to click Learn More and see the output records of a transaction.

Reply:
Good point, I left if empty because it `TransactionHistoryEntry` didn't have it but no reason not to add it.